usr/share/rear/restore/NSR/default/400_restore_with_nsr.sh

```diff
@@ -15,15 +17,26 @@ if is_true "$NSR_CLIENT_MODE"; then
     rear_shell "Has the restore been completed and are You ready to continue the recovery?"
 else
     LogUserOutput "Starting nsrwatch on console 8"
-    TERM=linux nsrwatch -p 1 -s $(cat $VAR_DIR/recovery/nsr_server ) </dev/tty8 >/dev/tty8 &
+    TERM=linux nsrwatch -p 1 -s $(cat $VAR_DIR/recovery/nsr_server) </dev/tty8 >/dev/tty8 &
```

Comment:
Only by the way as I see it (not related to this pull request topic):
Does a "console 8" i.e. /dev/tty8 always exist and is it always usable?
I think that may depend on kernel command line parameters
and USE_SERIAL_CONSOLE settings and things like that.

Reply:
I suppose @gdha can tell You about it since he implemented this entire NSR workflow (?). I just touched it removing the blank. 

BTW, the `nsrwatch -p 1 ...` is not working (any more). During my tests the `nsrwatch` never was started whereas leaving the `-p 1` out is working perfectly fine. (using a NSR client version 8.4.2)

Reply:
Then just remove the `-p 1` with some comment
that it is no longer working at least since NSR client version 8.4.2

Reply:
As far as I know (but I am not at all a TTY expert) with
```
if tty -s </dev/tty8 ; then ...
```
one can test if /dev/tty8 is a tty and I think this is the condition
that muts be fulfilled to run `nsrwatch ... </dev/tty8 >/dev/tty8`

Reply:
I removed the `-p 1` parameter from the `nsrwatch` command.

Reply:
@hpannenb NSR workflow implementation was done a very long time ago (8 years approx.) and I never used it since then (it was implemented on consultancy basis). Therefore, you are the master for this NSR workflow for the moment.

Reply:
I think it is OK to run `nsrwatch ... </dev/tty8 >/dev/tty8 &`
without a test like `if tty -s </dev/tty8 ; then ...`
because I think `nsrwatch` is only a nice to have thing and if that fails
the actual restore with the NSR `recover` command is not affected.